src/supporting-data/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,280 @@
+import cn from 'classnames';
+import { Button } from 'franklin-sites';
+import { ChangeEvent, useId } from 'react';
+import { Link } from 'react-router-dom';
+
+import { Location, LocationToPath } from '../../app/config/urls';
+import { useFormLogic } from '../../contact/adapters/contactFormAdapter';
+import SupportingDataIllustration from '../../images/Supporting_data_illustration.jpg';
+import HTMLHead from '../../shared/components/HTMLHead';
+import { SingleColumnLayout } from '../../shared/components/layouts/SingleColumnLayout';
+import styles from './styles/landing-page.module.scss';
+
+const metaDescription =
+  'The Supporting Data section enhances UniProt by providing curated and automated datasets that enrich protein entries with detailed annotations, including keywords, localization, functions, taxonomy, and literature citations.';
+
+const LandingPage = () => {
+  const formId = useId();
+
+  const { handleSubmit, handleChange, sending } = useFormLogic();
+
+  const handleTextareaChange = (e: ChangeEvent<HTMLTextAreaElement>) => {
+    e.target.setCustomValidity(
+      e.target.value.trim().length >= 3
+        ? ''
+        : 'Please enter more details about the database'
+    );
+  };
+
+  return (
+    <SingleColumnLayout>
+      <div className={styles['landing-page']}>
+        <HTMLHead title="Supporting data">
+          <meta name="description" content={metaDescription} />
+        </HTMLHead>
+        <section className="uniprot-grid">
+          <h1 className="uniprot-grid-cell--span-12">Supporting Data</h1>
+          <div
+            className={cn(
+              'uniprot-grid-cell--small-span-12',
+              'uniprot-grid-cell--medium-span-4',
+              styles['image-container']
+            )}
+          >
+            <img
+              src={SupportingDataIllustration}
+              width={300}
+              height={300}
+              alt=""
+            />
+          </div>
+          <div className="uniprot-grid-cell--small-span-12 uniprot-grid-cell--medium-span-8">
+            The Supporting Data section enhances the functionality of UniProt by
+            providing curated and automated datasets that enrich protein entries
+            with detailed information. This section covers keywords, subcellular
+            localization, automatic annotations, functional insights, taxonomy
+            and literature citations. By integrating these data points, it
+            allows researchers to explore proteins more comprehensively,
+            uncovering their roles, interactions, and evolutionary significance.
+            The Supporting Data section serves as a vital resource for making
+            UniProt entries more informative and accessible for biological and
+            biomedical research.
+          </div>
+
+          {/* Categories */}
+          <h2 className="uniprot-grid-cell--span-12">Key Categories</h2>
```

Comment:
It seems we describe all of the namespaces while the word "key" makes me think it's a subset. I am also thinking that Datasets or Databases might be a little closer each of the namespaces.

Reply:
Datasets?